### PR TITLE
#181 fix time select not considering the locale correctly when setting the selected time

### PR DESCRIPTION
**Pull request by rezk2ll**
demo: https://streamable.com/q8npww

### CODE REVIEW

**Review by asmaNode — DISMISSED** [review]

**Review by tuanlc — DISMISSED** [review]
No test for these changes :wink:

**Comment by rezk2ll** [comment]
> No test for these changes

done

**Review by tuanlc — DISMISSED** [review]

File: src/esn.calendar.libs/app/components/time-select/time-select.controller.spec.js [hunk 1]
```diff
@@ -194,6 +194,20 @@ describe('the calTimeSelectController', function() {
       ctrl.onSelectingTimeOption('13:00');
       expect(ctrl.onTimeChange).to.have.been.called;
     });
+
+    it('should call the onTimeChange binding after a date change', function() {
```
Comment (huy-ta) [inline src/esn.calendar.libs/app/components/time-select/time-select.controller.spec.js:198]: The test case's name seems wrong.

Comment (rezk2ll) [reply src/esn.calendar.libs/app/components/time-select/time-select.controller.spec.js]: done, thanks

**Review by tuanlc — APPROVED** [review]

**Review by asmaNode — APPROVED** [review]

**Pull request merged**